Makefile

```diff
@@ -108,9 +108,8 @@ testScripts: # Run e2e test suite
 	@forge script script/external/DeployFeeManager_v1.s.sol
 
 	## factories
-	@forge script script/factories/DeployModuleFactory_v1.s.sol
-	
-	## @note Because the Beacon doesnt allow to be passed addresses without implementations we cant test the script properly anymore without breaking the script testing structure we used up until now.
+	## @note Because the Dependecies on other contracts doesnt allow to be passed addresses without implementations we cant test the script properly anymore without breaking the script testing structure we used up until now.
```

Comment:
Is there something we need to do/can do here? Otherwise this comment doesn't really do/help much?

Reply:
Yeah Pablo is tackling that in the script revamp. In the talk on friday we agreed that we would straightup remove them from the make tests, as they are basically tested via the deploymentscript anyway